doc/start/documenting-ceph.rst

```diff
@@ -46,7 +45,8 @@ Making Contributions
 
 Making a documentation contribution involves the same basic procedure as making
 a code contribution, with one exception: you must build documentation source
-instead of compiling program source. This sequence (the sequence of building the documentation source) includes the following steps:
+instead of compiling program source. This sequence (the sequence of building
+the documentation source) includes the following steps:
```

Comment:
Suggest we remove entirely mention of building the docs, it serves only to frighten people.  Jenkins does that for us, and I doubt that _anyone_ actually does it locally.

Reply:
_I_ build the docs locally for almost every commit I make. I take your point, here though. I might move the build-the-docs material to a build-the-docs-if-you-really-want-to section somewhere off to the side, but I'm not going to do the work on that today. I'll make a note to myself to revisit it, though, and I think in the long run we will remove this (very old) reference to the difference between building the docs and compiling source code.